.jsdoc.js

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
2017!

Reply:
Oh man! I will patch this in [toolkit](https://github.com/googleapis/toolkit).